Flow.Launcher/Helper/HotKeyMapper.cs

```diff
@@ -7,97 +7,96 @@
 using System.Windows;
 using Flow.Launcher.ViewModel;
 
-namespace Flow.Launcher.Helper
+namespace Flow.Launcher.Helper;
+
+internal static class HotKeyMapper
 {
-    internal static class HotKeyMapper
+    private static Settings _settings;
+    private static MainViewModel _mainViewModel;
```

Comment:
Same renaming as with `_uri` a few files above. Also should be safe because these fields are `private`.